docs/content/en/docs/reference/configuration/tests.md

```diff
@@ -5,14 +5,128 @@ weight: 5
 aliases: ["/docs/reference/tests"]
 ---
 
-Vector enables you to unit test [transforms] in your processing topology
+Vector enables you to unit test [transforms] in your processing topology. The goal of unit tests in
+general is to improve the maintainability of configurations containing larger and more complex
+combinations of transforms.
 
-You can define unit tests in your Vector configuration file to cover a network of transforms within the topology. The intention of these tests is to improve the maintainability of configurations containing larger and more complex combinations of transforms.
+Unit tests in Vector work just like unit tests for standard software libraries:
+
+1. Provide a set of **inputs** to a transform (or a network of transforms)
+1. Provide expected **outputs** from the changes made by the transform(s)
+1. Receive directly actionable feedback on test failures
+
+If *any* of your unit tests fail, the Vector instance doesn't start up. This makes unit tests a
```

Comment:
This is another odd one to me as I don't think of any place were unit tests are run prior to startup - we have at least one place where `validate` is run though.

Reply:
Agreed, unit tests don't run as part of Vector start-up.